toolkit/tools/internal/safechroot/safechroot_test.go

```diff
@@ -71,7 +71,7 @@ func TestInitializeShouldCreateRoot(t *testing.T) {
 	if buildpipeline.IsRegularBuild() {
 		assert.True(t, !os.IsNotExist(err))
 	} else {
-		assert.True(t, !os.IsNotExist(err))
+		assert.True(t, os.IsNotExist(err))
```

Comment:
Looks like we expect err=1 for non-regular build which implies we support only regular builds. Should we assert IsRegularBuild as well?

Reply:
@kanikanema:
In my opinion `err` is a type; and internally calls [underlyingError](https://cs.opensource.google/go/go/+/refs/tags/go1.19:src/os/error.go;l=133;drc=0f6ee42fe063a48d7825bc03097bbb714aafdb7d;bpv=0;bpt=1).
In anycase, could you elaborate a little more;

Reply:
@kanikanema on further inspection; it seems the original `if` condition was reduandant.

Buddy Build: [223399](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=223399&view=results)